airflow/contrib/utils/kubernetes_utils.py

```diff
@@ -179,37 +179,47 @@ def dict_to_env(source, task_instance, context=None):
     return retval
 
 
-def uniquify_job_name(task_instance, context, run_timestamp=None, job_name=None):
+def uniquify_job_name(task, context, job_name=None):
```

Comment:
I may be thinking of this wrong, but `task` doesn't change for DAGs that run more than once right? Is this an issue?

Reply:
If a DAG runs every 30 min, it will have the same `task` name is what I mean, right ?

Reply:
It's sneakily obscured by the way the diff highlights, but the execution date actually is a part of the task name - see line 203.